client/my-sites/comments/comment/style.scss

```diff
@@ -0,0 +1,163 @@
+/** @format */
+// Comment Base Style
+.card.comment {
+	font-size: 14px;
+	margin: 0 auto;
+	padding: 0;
+	transition: margin 0.15s linear;
+
+	.accessible-focus &:focus {
+		box-shadow: 0 0 0 1px $blue-medium, 0 0 0 3px $blue-light;
+		z-index: z-index('root', '.comment-detail.card.accessible-focus:focus');
+	}
+}
+
+// Comment Header Block
+.comment__header {
+	cursor: pointer;
+	display: flex;
+	flex-flow: row;
+	flex-wrap: nowrap;
+
+	.comment__author {
+		padding: 8px;
+	}
+}
+
+.comment__bulk-select {
+	align-self: center;
+	padding-left: 16px;
+
+	.form-checkbox {
+		margin: 0;
+	}
+}
+
+.button.comment__toggle-expanded {
+	padding-left: 16px;
+	padding-right: 16px;
+
+	.gridicon {
+		fill: $gray;
+		transform: rotate(0deg);
+		transition: transform 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275), color 0.2s ease-in;
+	}
+
+	&:hover .gridicon {
+		fill: $blue-medium;
+	}
+}
+
+// Comment Author Block
+.comment__author {
+	display: flex;
+	flex-flow: row;
+	flex-grow: 1;
+	flex-wrap: nowrap;
+	width: 0;
+}
+.comment__author-avatar {
+	align-self: center;
+	height: 32px;
+	flex-grow: 0;
+	flex-shrink: 0;
+	padding: 0 8px;
+	width: 32px;
+}
+.comment__author-info {
+	display: flex;
+	flex-flow: column;
+	width: calc(100% - 48px);
+}
+.comment__author-info-element {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+
+	> div {
```

Comment:
Might be a better fit for picking an inline element like <span>, or looking into some flex options.

Reply:
Awesome catch!